ui-speedspacechart/src/components/const.ts

```diff
@@ -10,12 +10,20 @@ export const MARGINS = {
   ELECTRICAL_PROFILES_MARGIN_TOP: 8,
 };
 
-export const LAYERS_HEIGHTS = {
+export const LINEAR_LAYERS_HEIGHTS = {
   ELECTRICAL_PROFILES_HEIGHT: 56,
   POWER_RESTRICTIONS_HEIGHT: 40,
   SPEED_LIMIT_TAGS_HEIGHT: 40,
 };
 
+export const LinearLayerHeightsByName = {
```

Comment:
This variable name should be UPPER_CASE_SNAKE_CASE instead of PascalCase for consistency (it actually bothered me when I was reading your modifications to `components/utils.ts`, it looked like you were iterating on the keys of a type).

Reply:
Done